hubmapconsortium/ingest-pipeline
# Jpuerto psc/globus auth

What I Did:
Added functionality to lock Airflow behind an authentication routine. The authentication routine mirrors the routine that is used for other HubMap resources. That is, Oauth2 authentication with Globus.

Why I Did It:
This was done to allow external users to access Airflow resources. This also allows the resources to use the same user list as the HubMap resources.

How to Test:
Go to https://hivevm191.bridges.psc.edu:5555
From here, you should be redirected automatically to the Globus log-in page to authenticate with your institution's information.
After entering your information, you should be redirected back to the Airflow dashboard.

Patch Notes:
This change adds a few pip packages to the requirements.txt file.
This change also adds some configuration variables to the airflow.cfg file that MUST be used to be able to run appropriately.
